docs/faq/perf.md

```diff
@@ -117,37 +117,46 @@ and  [MXNet-1.2.0.rc1](https://github.com/apache/incubator-mxnet/releases/downlo
 
 - K80 (single GPU)
 
-| Batch | Alexnet | VGG    | Inception-BN | Inception-v3 | Resnet 50 | Resnet 152 |
+| Batch | Alexnet | VGG 16    | Inception-BN | Inception-v3 | Resnet 50 | Resnet 152 |
 |-------|---------|--------|--------------|--------------|-----------|------------|
 | 1     | 243.93  | 43.59  | 68.62        | 35.52        | 67.41     | 23.65      |
 | 2     | 338.16  | 49.14  | 113.41       | 56.29        | 93.35     | 33.88      |
 | 4     | 478.92  | 53.44  | 159.61       | 74.43        | 119.18    | 45.23      |
 | 8     | 683.52  | 70.50  | 190.49       | 86.23        | 131.32    | 50.54      |
 | 16    | 1004.66 | 109.01 | 254.20       | 105.70       | 155.40    | 62.55      |
 | 32    | 1238.55 | 114.98 | 285.49       | 116.79       | 159.42    | 64.99      |
+| 64 | 1346.72 | 123.56 | 308.73 | 122.21 | 167.58 | 70.21 |
+| 128 | 1416.91 | OOM | 320.98 | 123.11 | 171.55 | 71.85 |
```

Comment:
Is it still OOM after this PR? https://github.com/apache/incubator-mxnet/pull/10847

Reply:
yes. can you give it a try?